plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -69,5 +86,18 @@
     </form>
     {% endif %}
 {% endif %}
+
+{% if javascripts.external|length > 0 %}
+    {% for script in javascripts.external %}
+        <script type="text/javascript" src="{{ script|raw }}"></script>
+    {% endfor %}
+{% endif %}
+{% if javascripts.inline|length > 0 %}
+    <script>
+        {% for script in javascripts.inline %}
+        {{ script|raw }}
```

Comment:
The `|raw` here could cause security issues if `script` contains any user input (same for StyleSheets). I presume there's no other way to do that as `|e('js')` would only escape a JavaScript string. If we wouldn't need inline scripts / stylesheets that would be great, otherwise we will need to document it at least in the method `addJavascript/addStylesheets` that all user input must be escaped before.